packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -296,9 +296,7 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
         // Hardcoded `scalingMode` since it is highly unlikely that CONTOSO would ever want to use a different scaling mode for screenshare.
         // Using `Crop` would crop the contents of screenshare and `Stretch` would warp it.
         // `Fit` is the only mode that maintains the integrity of the screen being shared.
-        createViewResult = await callClient.createView(call.id, participant.identifier, screenShareStream, {
-          scalingMode: 'Fit'
-        });
+        await callClient.createView(call.id, participant.identifier, screenShareStream, { scalingMode: 'Fit' });
```

Comment:
Why do we change this line? I guess the view result is still useful to manage screenShareStream life cycle

Reply:
When a remote participant is screen sharing, their video stream view gets overridden here.
For now, the priority is to use their video stream view.
We need to find a better way to consume both stream and screen share view objects

Reply:
Yea, I can see some concerns to make it easier here
This is a public facing api, but we change the behavior a bit here, I would still consider this is a behavior breaking change from our side(although I doubt if there is anyone use this, but let's not lie to breaking change definition)

One way I can think of is not changing this, but add an if-condition in parent caller - it's easy to determine if a stream is a remote video stream or just a screen share stream - if it is a screen share stream, don't override it

Reply:
BTW, @JamesBurnside do you remember why do we add a return value here, I guess there is a final consumer of this return value, but I just can't find it in the code

Reply:
Removed this change from this PR
This will be coming along with it's usage in another PR